lynx.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Vikas(Vikas_2_0)",
+			"oem": "Google",
+			"device": "Pixel 7a",
+			"filename": "crDroidAndroid-15.0-20241106-lynx-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lynx/11.x/crDroidAndroid-15.0-20241106-lynx-v11.0-BETA.zip/download",
+			"timestamp": 1730886923,
+			"md5": "ac172d6c5f14f3b314c627cfa65eba56",
+			"sha256": "08064f559c005a4965d5225d6612e82ac63b651c96269016e2349d006a459446",
+			"size": 1591510367,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-official-lynx-15-0-crdroidandroid-v11-x.4702075/",
```

Comment:
as has been discussed few times already, please update forum post as per latest template
footer is bad